frontend/src/mainApp.js

```diff
@@ -15,14 +15,12 @@ function App() {
   return (
     <div>
       <Switch>
-        <Route exact path="/CreateCard">
-          <CreateCard></CreateCard>
-        </Route>
-        <Route exact path="/MyFolders" component={MyFolders} />
-        <Route exact path="/ImageInterface" component={ImageInterface} />
-        <Route exact path="/YoutubeInterface" component={YoutubeDetect} />
-        <Route exact path="/CreateFolder" component={CreateFolder} />
-        <Route exact path="/InsideFolder" component={InsideFolder} />
+        <Route exact path="/createCard" component={CreateCard} />
```

Comment:
These new paths don't match the design doc.

Reply:
Changed to match design doc

Reply:
done